packages/web-app-files/src/components/FilesList/ResourceTiles.vue

```diff
@@ -287,6 +289,9 @@ export default defineComponent({
     return {
       ImageDimension
     }
+  },
+  computed: {
+    ...mapState('Files', ['areFileExtensionsShown'])
   }
```

Comment:
You can do that via Composition API as well: `store.getters['Files/areFileExtensionsShown']` 🙂

Reply:
Applied, good point

Reply:
Ahh sorry, apparently there is no `getter`, so you need to go via the `state`: `store.state.Files.areFileExtensionsShown`.

And you need to move it into a computed to be reactive then.

Reply:
Now we should be good